docs/reference/lifecycle-hooks/SlackNotifierHook.rst

```diff
@@ -0,0 +1,13 @@
+==========================

```

Comment:
and then this file needs to be referenced in an index.rst/md file for the docs to build.